client/e2e/src/add-note.e2e-spec.ts

```diff
@@ -0,0 +1,41 @@
+import { AddNotePage, TestNote } from './add-note.po';
+import { by, element, browser, protractor } from 'protractor';
+import { E2EUtil } from './e2e.util';
+
+
+describe('Add note:', () => {
```

Comment:
Low-key, I'm so happy that you all used the same capitalization and punctuation as the other e2e tests 🙃 That's wonderful!